float64 int1
float64 int2
---
float64 sum
